src/supertux/game_object_factory.cpp

```diff
@@ -224,10 +224,11 @@ GameObjectFactory::init_factories()
   add_factory<Snail>("snail", OBJ_PARAM_PORTABLE | OBJ_PARAM_DISPENSABLE);
   add_factory<SnowBall>("snowball", OBJ_PARAM_DISPENSABLE);
   add_factory<Snowman>("snowman", OBJ_PARAM_DISPENSABLE);
-  add_factory<SpiderMite>("spidermite", OBJ_PARAM_DISPENSABLE);
   add_factory<Spiky>("spiky", OBJ_PARAM_DISPENSABLE);
   add_factory<Stalactite>("stalactite", OBJ_PARAM_DISPENSABLE);
   add_factory<Stumpy>("stumpy", OBJ_PARAM_DISPENSABLE);
+  add_factory<Tarantula>("spidermite", OBJ_PARAM_DISPENSABLE); // Backward compatibilty.
```

Comment:
Remove `OBJ_PARAM_DISPENSABLE` from here, so it doesn't show up twice in the dispenser object select menu.

```suggestion
  add_factory<Tarantula>("spidermite"); // Backward compatibilty.
```